draft-regext-rdap-extensions.md

```diff
@@ -514,9 +514,9 @@ A> issue #52
 
 Usage of a bare extension identifier conflicts with the guidance in
 [@!RFC9083, section 2.1].  This document clarifies [@!RFC9083, Section 2.1] to explicitly allow
-this pattern but advises extension authors to use this pattern when only one
+this pattern but extension authors MUST NOT use this pattern when only one
```

Comment:
How about:

Usage of a bare extension identifier conflicts with the guidance in [@!RFC9083, section 2.1]. Previously, extension authors have used this pattern when only one query path, JSON name, and/or object class is being defined by the extension. Henceforth, this pattern MUST NOT be used.